autoinstallation/package/agama-auto.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Thu Feb 13 12:10:11 UTC 2025 - Knut Anderssen <kanderssen@suse.com>
+
+- Adapted agama-auto script to reboot by default once the unattended
+  installation is finished allowing to stop avoid it through the
```

Comment:
This long phrase is confusing. I guess you wanted to write "stop" or "avoid," not both.

Reply:
When you rewrite two or three time a sentence and it is clear in your mind but you do not check the resultant one...